src/app/cities-search/cities-search.component.html

```diff
@@ -0,0 +1,17 @@
+<div>
+  <label
+    ><b>{{ label }}</b></label
+  >
+  <mat-form-field class="example-full-width" appearance="outline">
+    <input type="text" [placeholder]="placeholder" matInput [formControl]="formControl" [matAutocomplete]="auto" />
+    <mat-autocomplete
+      #auto="matAutocomplete"
+      [displayWith]="displayOption"
+      (optionSelected)="onSelected($event.option.value)"
+    >
+      <mat-option *ngFor="let option of options" [value]="option">
+        {{ displayOption(option) }}
```

Comment:
Such function calls are going to be called a lot more often in Angular then in React. A better solution would to change it into pipe `{{option | displayOption:arg }}`
See: https://angular.io/guide/pipes#example-transforming-a-value-exponentially

Reply:
👍